uat/integrations/README.md

```diff
@@ -0,0 +1,32 @@
+# Test Kubeflow Integrations
+
+Run Python notebooks with Pytest inside a Notebook Server to verify the integration of Kubeflow
+with different components. The following tests are currently provided:
+* [Test Katib Integration](examples/katib-integration.ipynb)
+* [Test KFP Integration](examples/kfp-integration.ipynb)
+* [Test MinIO Integration](examples/minio-integration.ipynb)
+* [Test MLFlow Integration](examples/mlflow-integration.ipynb)
+
```

Comment:
Subjective, minor comment: instead of listing the notebooks, what about just pointing to their directory?  That way when someone adds a 5th and forgets this chore, we aren't out of date.

Reply:
It's unfortunate that even I forgot to update the list when I added the Katib notebook😅
I'm not sure how I feel about this. I think I like listing all of the included tests here. The point is that we might have more complicated scenarios that might need a description here, apart from the fact that it's nice looking at a single place to find the available tests.

Reply:
yeah in the perfect world I like this list.  In the world where I live and make mistakes, I don't haha

A notebooks directory is also a good place to summarise everything, and each `.ipynb` file I think is self-documenting (or at least, should be).  So this all feels redundant and chore-like

Reply:
Dropped!